.github/workflows/ubuntu.yml

```
@@ -141,6 +150,12 @@ jobs:
       # xsel for clipboard tests
       run: sudo apt-get update && sudo apt-get install -y xsel ${{ env.EXTRA_APT }}
 
+    - name: Generate extra locales
+      # These extra locales will be available for locale.setlocale() calls in tests
+      run: |
+        sudo locale-gen ${{ env.EXTRA_LOC }}
+      if: ${{ env.EXTRA_LOC != '' }}
```

```suggestion
      if: ${{ matrix.extra_loc == 'zh_CN' || matrix.extra_loc == 'it_IT' }}
```

Just to eliminate another env variable

---

Hi @mroeschke , I think I understood your intent, but I think the actual changes needed are a bit different, so let me know if what I push is ok.